src/github.rs

```diff
@@ -166,6 +166,33 @@ impl GithubClient {
         let (body, _req_dbg) = self.send_req(req).await?;
         Ok(serde_json::from_slice(&body)?)
     }
+
+    pub(crate) async fn new_issue(
+        &self,
+        repo: &IssueRepository,
+        title: &str,
+        body: &str,
+        labels: Vec<String>,
+    ) -> anyhow::Result<NewIssueResponse> {
+        #[derive(serde::Serialize)]
+        struct NewIssue<'a> {
+            title: &'a str,
+            body: &'a str,
+            labels: Vec<String>,
+        }
+        self.json(self.client.post(repo.url(&self)).json(&NewIssue {
```

Comment:
The URL for this doesn't look correct to me. It needs to go to the `/issues` endpoint.

Also, the `post` method needs to be on `self` to properly configure the request.

```suggestion
        let url = format!("{}/issues", repo.url(&self));
        self.json(self.post(&url).json(&NewIssue {
```

Reply:
Yeah, good catch.